project/Mima.scala

```diff
@@ -76,6 +76,8 @@ object Mima {
     ProblemFilters.exclude[DirectMissingMethodProblem](
       "scala.meta.Defn#Given#DefnGivenImpl.copy$default$6"
     ),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Defn#Given#DefnGivenImpl.this")
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Defn#Given#DefnGivenImpl.this"),
```

Comment:
what was the problem with mima? adding a method causes this? or did we change the signature?

Reply:
This happens when adding a methods for public traits. However since no one really can sensibly extend CaseTree, I think it's not a problem.